mysql/datadog_checks/mysql/statements.py

```diff
@@ -148,10 +148,13 @@ def _query_summary_per_statement(self, db):
 
     def _normalize_queries(self, rows):
         normalized_rows = []
+        obfuscate_options = json.dumps(
```

Comment:
The options can never change once the check instance is initialized so we only need to do this once for the lifetime of the check.

Reply:
Updated: https://github.com/DataDog/integrations-core/pull/9639/commits/ca7de48a9470f09cffa1c0787878c9e46e9e8e78